Src/WitsmlExplorer.Frontend/components/Sidebar/FilterPanel.tsx

```diff
@@ -26,111 +46,128 @@ const FilterPanel = (): React.ReactElement => {
-            value={curveThreshold.timeInMinutes}
-            autoComplete={"off"}
-          />
-          <FormControlLabel
-            control={
-              <Checkbox
-                id="curveThreshold-hideInactive"
-                onChange={(event) =>
-                  dispatchNavigation({ type: NavigationType.SetCurveThreshold, payload: { curveThreshold: { ...curveThreshold, hideInactiveCurves: event.target.checked } } })
-                }
-                value={curveThreshold}
-              />
-            }
-            label={"Hide inactive time curves"}
-          />
+          <div style={{ paddingTop: "0.75rem" }}>
+            <Checkbox
+              id="filter-isActive"
+              onChange={(event) => dispatchNavigation({ type: NavigationType.SetFilter, payload: { filter: { ...filter, isActive: event.target.checked } } })}
+              value={"Show active Wells / Wellbores"}
+              color={"primary"}
+              style={{ height: "0.625rem", userSelect: "none" }}
+              label={"Show active Wells / Wellbores"}
+            />
+          </div>
+          <div style={{ userSelect: "none" }}>
+            <Checkbox
+              id="filter-objectGrowing"
+              onChange={(event) => dispatchNavigation({ type: NavigationType.SetFilter, payload: { filter: { ...filter, objectGrowing: event.target.checked } } })}
+              value={"Show growing logs"}
+              color={"primary"}
+              label={"Show growing logs"}
+              style={{ userSelect: "none" }}
+            />
+          </div>
+          <div
+            style={{
+              display: "grid",
+              gridTemplateColumns: "1fr 80px",
+              alignItems: "center",
+              paddingBottom: "10px",
+              paddingLeft: "0.8rem",
+              paddingRight: "0.5rem",
+              userSelect: "none"
+            }}
+          >
+            <span style={{ display: "flex", gap: "0.5rem" }}>
+              <Typography token={{ fontSize: "0.75rem", fontFamily: "EquinorMedium", color: colors.text.staticIconsTertiary }} style={{ display: "flex", gap: "7px" }}>
+                Set threshold for time curve
+              </Typography>
+              <Typography token={{ fontStyle: "italic", fontFamily: "EquinorRegular", fontSize: "0.75rem", color: colors.text.staticIconsTertiary }}> (minutes) </Typography>
+            </span>
+
+            <TextField
+              id="curveThreshold-time"
+              type="number"
+              min={0}
+              onChange={(event: ChangeEvent<HTMLInputElement>) =>
+                dispatchNavigation({ type: NavigationType.SetCurveThreshold, payload: { curveThreshold: { ...curveThreshold, timeInMinutes: Number(event.target.value) } } })
+              }
+              value={curveThreshold.timeInMinutes}
+              autoComplete={"off"}
+            />
+          </div>
+
+          <div style={{ userSelect: "none" }}>
+            <Checkbox
```

Comment:
The checkbox becomes unchecked after hiding the FilterPanel, even though hideInactiveCurves is set to true.

Reply:
updated